scrapers/wowgirlsblog.yml

```diff
@@ -0,0 +1,33 @@
+name: "WOW Girls Blog"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wowgirlsblog.com
+
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1
+      Date:
+        selector: //meta[@property="article:published_time"]/@content
+        postProcess:
+          - replace:
+            - regex: '([^ ]+)T.*'
+              with: $1
+      Details: //meta[@property="og:description"]/@content
```

Comment:
Does the description from the meta contain any useful info? For most scenes it is just the performers and the title of the scene ?

Reply:
I don't know. But you can disable to import the description if you don't like it. Otherwise you will not recognize useful information.